userspace/libscap/engine/bpf/scap_bpf.c

```diff
@@ -1504,20 +1504,33 @@ int32_t scap_bpf_load(
 			fp = fopen(filename, "r");
 			if(fp == NULL)
 			{
-				snprintf(handle->m_lasterr, SCAP_LASTERR_SIZE, "can't open %s: %s", filename, scap_strerror_r(buf, errno));
-				return SCAP_FAILURE;
+				// When missing NUMA properties, CPUs do not expose online information.
+				// Fallback at considering them online if we can at least reach their folder.
+				// This is useful for example for raspPi devices.
+				// See: https://github.com/kubernetes/kubernetes/issues/95039
+				snprintf(filename, sizeof(filename), "/sys/devices/system/cpu/cpu%d/", j);
+				if (access(filename, F_OK) == 0)
+				{
+					online = 1;
+				}
+				else
+				{
+					snprintf(handle->m_lasterr, SCAP_LASTERR_SIZE, "can't open %sonline: %s", filename, scap_strerror_r(buf, errno));
```

Comment:
```suggestion
					snprintf(handle->m_lasterr, SCAP_LASTERR_SIZE, "cannot access '%s' file: %s", filename, scap_strerror_r(buf, errno));
```

Reply:
`%sonline` is misleading and is not the actual file we are looking for here, so I would remove it... just proposed a rephrase but feel free to ignore it

Reply:
Mmmh if both `/sys/devices/system/cpu/cpu%d/online` and `/sys/devices/system/cpu/cpu%d/` cannot be reached, i'd like to notify the user about the former.